docs/troubleshooting.md

```diff
@@ -0,0 +1,22 @@
+# How to troubleshoot issues
+
+Sometimes Ollama may not perform as you expect. One of the best ways to figure out what just happened is to take a look at the logs. You can find the logs on Mac by running the command:
+
+```shell
+cat ~/.ollama/logs/server.log
```

Comment:
or `tail -f`?

Reply:
Often the error is spread across a number of lines and having to run this a few times expanding the range, and explain the cli args to do so, is harder than just doing a single cat. But thanks for the comment